src/governance.ts

```diff
@@ -208,6 +213,50 @@ export class GovernanceCanister {
     return proposalInfo ? toProposalInfo(proposalInfo) : undefined;
   };
 
+  /**
+   *
+   * Registers vote for a proposal from the neuron passed
+   *
+   * @param neuronId: NeuronID
+   * @param vote: Vote
+   * @param ProposalId: ProposalId
+   * @returns EmtpyResponse
+   */
+  public registerVote = async ({
+    neuronId,
+    vote,
+    proposalId,
+  }: {
+    neuronId: NeuronId;
+    vote: Vote;
+    proposalId: ProposalId;
+  }): Promise<EmptyResponse> => {
```

Comment:
Not related to your PR, I guess it already exists but isn't `EmptyResponse` a bit misleading?

It's a variant that contains either Ok or an Err.

Being said, if you agree and have an idea of name, we can refactor the name in another PR, probably better.

Reply:
A little misleading yes, I was expecting more like `undefined` or something like that.

I don't have a better name, that that will be easy to change. The patter of `Ok` and `Error` is the same as we use in other methods, so that's good for me.

Reply:
Thanks for the feedback, I see. I'll think about it.